solcast/pv_power_sites.py

```diff
@@ -0,0 +1,125 @@
+from .api import Client, Response
+from .urls import base_url, pv_power_site, pv_power_sites
+
+
+def list_pv_power_site(**kwargs) -> Response:
+    """List available PV power sites"""
+    client = Client(base_url=base_url, endpoint=pv_power_sites)
+
+    params = {"format": "json", **kwargs}
+
+    res = client.get(params)
+
+    return res
+
+
+def get_pv_power_site(resource_id: str, **kwargs) -> Response:
+    """
+    Get an existing PV power site's specifications
+
+    Args:
+        resource_id: unique string to identify the location that was generated when creating a site
+
+    """
+    client = Client(base_url=base_url, endpoint=pv_power_site)
+
+    params = {"resource_id": resource_id, "format": "json", **kwargs}
+
+    res = client.get(params)
+
+    return res
+
+
+def create_pv_power_site(
+    name: str, latitude: float, longitude: float, **kwargs
+) -> Response:
+    """
+    Create PV power site to be used with Solcast's advanced PV power model
+
+    Args:
+        name: arbitrary string to identify the location, unique not required
+        latitude: in decimal degrees, between -90 and 90, north is positive
+        longitude: in decimal degrees, between -180 and 180, east is positive
+        **kwargs:
+
+    See https://docs.solcast.com.au/ for full list of parameters.
+
+    """
+    client = Client(base_url=base_url, endpoint=pv_power_site)
+
+    params = {
+        "name": name,
+        "latitude": latitude,
+        "longitude": longitude,
+        "format": "json",
+        **kwargs,
+    }
+
+    res = client.post(params)
+
+    return res
+
+
+def patch_pv_power_site(resource_id: str, **kwargs) -> Response:
+    """
+    Patch an existing PV power site to partially update the site's specifications
+
+    Args:
+        resource_id: unique string to identify the location that is generated when creating a site
+        **kwargs:
+
+    See https://docs.solcast.com.au/ for full list of parameters.
```

Comment:
Formatting is a bit weird - should this sentence be in line with **kwargs:?

Reply:
Added these to all functions